pkg/controller/workload/workload_event.go

```diff
@@ -62,6 +63,20 @@ type Endpoint struct {
 	portList    []*workloadapi.Port
 }
 
+type NetworkAddress struct {
+	Network string
+	Address uint32
+}
+
+type WorkloadStore struct {
+	ByUid  map[string]*workloadapi.Workload
```

Comment:
Why do we need indexer by uid? I cannot imagine

Reply:
when workload removed, only the uid of workload can be get, and address info can get by byuid[uid].